pybamm/input/parameters/lithium-ion/mechanicals/lico2_graphite_Ai2020/parameters.csv

```diff
@@ -0,0 +1,45 @@
+Name [units],Value,Reference,Notes
```

Comment:
Those capitalizations are the same as others. They were copied here actually.

This file is used to update a list of parameters, if an existing set doesn't have those parameters. I used this file for testing other cell parameters. In the new set (Ai2020), this is not necessary.

Reply:
What I mean is the capitalization of the third word, like in "Positive electrode Initial crack length". I know this is nitpicking.

Regarding testing other cell parameters, I see what you are doing now. Wasn't sure about it at first but I think it's ok. Just need to be careful if someone comes along and wants to use the cracking submodel with some other parameters (like Ecker) that they don't use the wrong parameters by accident

Reply:
I see where you are from. Those capitalizations are below the highlighted content. I have corrected them.

This file contains mechnical parameters which should work for other parameters (like Ecker) or update the values manually. This is a temporal way to add those mechanical parameters, and it will be modified when there is a better way to do this.